Jenkinsfile.groovy

```diff
@@ -4,10 +4,8 @@ pipeline {
 	agent any
 	environment {
 		// Begin of Config
-		def gitURL = 'https://github.com/Metrilus/MetriCam2'
-		def gitBranch = 'master'
+		def STAUS_CONTEXT = 'MetriCam2 CI'
```

Comment:
What's a "STAUS_CONTEXT"? Why is it all-caps, unlike all other variables.
-> goes into issue